client/components/PuzzleNavBar/PuzzleNavBar.jsx

```diff
@@ -25,7 +25,13 @@ export function PuzzleNavBar({puzzleNum, puzzleDesc}) {
               </Link>
               <Nav.Link href="/UserProfile">User Profile</Nav.Link>
               <Nav.Link onClick={handleShowDesc}>Puzzle Description</Nav.Link>
+              <Form>
+                <Form.Group className="timer">
+                  <Form.Label>Time: {minutes}:{seconds?.toString().padStart(2, '0') || "00"}</Form.Label>
+                </Form.Group>
+              </Form>
```

Comment:
Why is this a form??? Seems like a really weird way to handle it.

As an aside, the timer looks a little odd being in the middle. Seems like it would be better over by the Log Out button on the right side. (also the bolding is odd)